### PR TITLE
Fix reviewlog descriptions by directly using activity log short string

### LINKED ISSUE #6602: Add-on Review Log should show action description for content approve/reject actions
Originally reported in https://github.com/mozilla/addons-server/issues/6310#issuecomment-334395413 but this deserve its own issue cause I'd like to fix it by doing some refactoring to avoid the issue for future actions as well.

STR:
- Approve or reject content for an add-on
- Go to review log, see your action

Expected Result:
- You should see what kind of action you did like "Content approved" or "Content Rejected"

Actual Result:
- You see only the add-on name and version in the "Event" column for those actions.

### PR DESCRIPTION
**Pull request by diox**
This allows us to stop having to hardcode descriptions in the reviewlog view, which was annoying since it's easy to forget...

Fix #6602

### CODE REVIEW

**Comment by diox** [comment]
@wagnerand is this ok ? This changes the descriptions to use the `short` property for each action instead of harcoding a string in the view, making us more future-proof and avoiding duplication. The only drawback is that it changes the string displayed a little, but they should still be meaningful.

**Comment by wagnerand** [comment]
@diox sounds good to me. The only ask I'd have is that we change `COMMENT_VERSION.short` to `_(u'Commented')` (from `_(u'Comment')`) in https://github.com/wagnerand/addons-server/blob/master/src/olympia/constants/activity.py#L215

File: src/olympia/editors/tests/test_views.py [hunk 1]
```diff
@@ -374,19 +374,35 @@ def test_request_info_logs(self):
...
         assert pq(r.content)('#log-listing tr td a').eq(1).text() == (
-            'needs super review')
+            'Super review requested')
 
     def test_comment_logs(self):
         self.make_an_approval(amo.LOG.COMMENT_VERSION)
         r = self.client.get(self.url)
         assert pq(r.content)('#log-listing tr td a').eq(1).text() == (
-            'commented')
+            'Comment')
```
Comment (wagnerand) [inline src/olympia/editors/tests/test_views.py]: My ask would also need this to be `Commented`.

File: src/olympia/editors/templates/editors/reviewlog.html [hunk 2]
```diff
@@ -46,13 +46,15 @@
                   {% else %}
                     {% set channel = 0 %}
                   {% endif %}
-                  {% if channel == amo.RELEASE_CHANNEL_LISTED %}
+                  {% if item.action == amo.LOG.REJECT_CONTENT.id or item.action == amo.LOG.APPROVE_CONTENT.id %}
```
Comment (eviljeff) [inline src/olympia/editors/templates/editors/reviewlog.html]: I guess this works until we start content reviewing unlisted add-ons, heh.

Comment (wagnerand) [reply src/olympia/editors/templates/editors/reviewlog.html]: We won't. Unlisted add-ons by design do not apply to content policies.

**Pull request merged**